docs/i2c_driver.md

```diff
@@ -235,3 +235,23 @@ Reads from a register on the I2C device.
 ### `i2c_status_t i2c_stop(void)`
 
 Stop the current I2C transaction.
+
+## I2C bitbang driver
+
+The I2C bitbang driver can be enabled on any targeted platform, with varying performance.
```

Comment:
…except `arm_atsam` (because the makefile parts are only for `avr` and `chibios`), but apparently everyone wants to forget that it even exists, so…

Reply:
Upon reading your comment, I also wanted to implement non-optimized atsam support, however it looks like atsam doesn't have the mcu-agnostic setPinOutput() and family defined, so I guess it's out-of-scope of this PR to implement that, so instead I will just edit the documentation to list only avr and chibios.